app/views/home/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1 class="text-blue-500">Hello, world!<h1>
\ No newline at end of file
+<div class="flex flex-col items-center px-10 md:px-32 lg:px-60 py-10 lg:py-14">
+  <h1 class="mb-7 text-primary">¡Hola!</h1>
+  <h5 class="mb-7">Te damos la bienvenida a</h5>
+
+  <form class="pb-12 w-full">
+    <div class="flex flex-col items-center">
+      <label class="pb-4 text-center">
+        Escribe tu dirección en la barra para ubicar los negocios cercanos
+      </label>
+      <input type="text" id="location" class="md:w-8/12 py-2 px-6 bg-blue-50 rounded-md font-light shadow" name="location" placeholder="Escribe tu dirección">
+    </div>
+  </form>
+  
+  <h3 class="mb-6 text-complementary">#QuédateEnCasa</h3>
+  <p class="text-center mb-3">
+    Tribo es una plataforma digital que te conecta con comercios y
+    vendedores cerca de ti y que pueden enviar a domicilio lo que necesitas de ellos.
+  </p>
+  <p class="md:w-6/12 text-center mb-8">
+    Sólo llama o envía un mensaje por WhatsApp
+    y recibe tu pedido en la puerta de tu hogar
+  </p>
+
+  <a href="" class="py-2 px-4 bg-primary hover:bg-white border-2 border-primary rounded-md text-white hover:text-primary font-semibold hover:shadow">Descarga la App</a>
```

Comment:
You can also use the Rails helper `link_to`, behaviour should be the same. Just FYI

Reply:
Is it useful to use `link_to`even if the link it will reference is not a route of the app but a link to another page or the Play Store? I thought `link_to` it was reserved for using alongside Rails Routing.

Reply:
The behaviour stays the same, but it helps by normalizing the code in `html.erb` files. But on the other side I like the concern separation by limiting the `link_to` to Rails routing :D

Reply:
So, no need to make the change here.